docs/image.md

```diff
@@ -142,6 +142,13 @@ img_ = mmcv.impad(img, (1000, 1200), pad_val=0)
 # pad the image to (1000, 1200) with different values for three channels.
 img_ = mmcv.impad(img, (1000, 1200), pad_val=[100, 50, 200])
 
+# pad the image on left, right, top, bottom borders with all zeros
+img_ = mmcv.impad(img, (10, 20, 30, 40), pad_val=0)
+
+# pad the image on left, right, top, bottom borders with different values 
+# for three channels.
+img_ = mmcv.impad(img, (10, 20, 30, 40), pad_val=[100, 50, 200])
```

Comment:
These examples are outdated.

Reply:
Thanks. It's fixed in the latest version.